### PR TITLE
CLI doc edits

### PR DESCRIPTION
**Pull request by james-haytko-sp**
Edit the CLI help files.

### CODE REVIEW

File: cmd/search/README.md
Comment (luke-hagar-sp) [inline cmd/search/README.md:1]: @james-haytko-sp Do you see value is keeping this file, and the other ones like it?
These files only exist in this repo, and I don't think they are referred to by anyone on a regular basis. I believe Jordan elected for us to refer everyone to the webdocs.

File: cmd/search/query.go [hunk 1]
```diff
@@ -15,8 +15,8 @@ func newQueryCmd() *cobra.Command {
 	var folderPath string
 	cmd := &cobra.Command{
 		Use:     "query",
-		Short:   "Manually Search using a specific Query and Indicies",
-		Long:    "\nRun a search query in IdentityNow using a specific Query and Indicies\n\n",
+		Short:   "Manually search using a specific query and indices",
```
Comment (luke-hagar-sp) [inline cmd/search/query.go:18]: Part of wanting to break out the markdown separate from the actual go code was a desire to make it easier for people like yourself to go in and make docs updates and changes.

And yet here you are going in anyway, finding all the right places, and making incredibly helpful updates directly to the cobra command structs.

@james-haytko-sp you are truly a legend, and I'm so glad you are on our team.

File: cmd/transform/create.go [hunk 2]
```diff
@@ -49,7 +49,7 @@ func newCreateCommand() *cobra.Command {
 			log.Debug("Transform", "transform", transform)
 
 			if transform.GetName() == "" {
-				return fmt.Errorf("the transform must have a name")
+				return fmt.Errorf("The transform must have a name")
```
Comment (luke-hagar-sp) [inline cmd/transform/create.go:52]: So many of these changes are amazing, and I love that you have gone through here so thoroughly.

In this case specifically go kinda throws a fit, they have a linting message that all error strings should start with a lowercase character

https://google.github.io/styleguide/go/decisions.html#error-strings

File: cmd/search/query.go (hunk 1 above)
Comment (james-haytko-sp) [reply cmd/search/query.go:18]: Thanks Luke! I appreciate it

File: cmd/search/README.md
Comment (james-haytko-sp) [reply cmd/search/README.md:1]: Ah yeah I think I went out of my way to make some changes that I wanted to make to the search doc on the developer site to make them consistent, but now I realize that we don't really need to maintain these readme files.

**Review by LukeHagar — APPROVED** [review]

**Review by luke-hagar-sp — APPROVED** [review]

**Pull request merged**